src/Func.cpp

```diff
@@ -162,7 +162,8 @@ void Func::define_extern(const std::string &function_name,
                          const std::vector<ExternFuncArgument> &args,
                          const std::vector<Type> &types,
                          int dimensionality,
-                         NameMangling mangling) {
+                         NameMangling mangling,
+                         bool uses_old_buffer_t) {
```

Comment:
Do we have to worry about unused variable warnings on some compilers? I'd typically write this:
    `bool /* uses_old_buffer_t */) {`